docs/reference/applications/interchain-account.mdx

```diff
@@ -40,15 +42,11 @@ flowchart TB
 
 Interchain Accounts allow you to make a remote call from **Chain A** to **Chain B** using the router (`InterchainAccountRouter`). Here's how it works:
 
-- We use [CREATE2](https://docs.openzeppelin.com/cli/2.8/deploying-with-create2) to compute the deterministic [OwnableMulticall](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/solidity/contracts/middleware/libs/OwnableMulticall.sol) contract address for you, which serves as a proxy for your cross-chain calls. You can explore this [here](#example-usage).
-
-- You can encode your call which includes the to address, call data, and the `msg.value` for each call, batched together in an array.
-
-- You send the encoded call to the **Chain A** router which gets relayed to the **Chain B** router.
-
-- After decoding the calls, the **Chain B** router checks if the computed address is already deployed or not. If not, we deploy the _OwnableMulticall_ contract.
-
-- The router then performs a multicall on the ICA address, which in turn makes the desired arbitrary call on **Chain B**.
+1. We use [CREATE2](https://docs.openzeppelin.com/cli/2.8/deploying-with-create2) to compute the deterministic [OwnableMulticall](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/solidity/contracts/middleware/libs/OwnableMulticall.sol) contract address for you, which serves as a proxy for your cross-chain calls. You can explore this [here](#example-usage).
```

Comment:
this technically happens in step 4
it might make most sense to just lift this from the numbered steps